gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -373,12 +345,10 @@ def __init__(self,
       if field not in header_fields.infos:
         raise ValueError('{} INFO not found in the header'.format(field))
       header_desc = header_fields.infos[field][_HeaderKeyConstants.DESC]
-      self._annotation_names_map[field] = _extract_annotation_names(
-          header_desc)
+      self._annotation_names_map[
```

Comment:
nit: I find it more readable to break on the `=` sign like:
```
self._annotation_names_map[field] = (
    annotation_parser.extract_annotation_names(header_desc))
```

Reply:
Done.
FYI, IntelliJ formatted it that way.